h2o-algos/src/main/java/water/rapids/prims/mungers/mungers/AstTargetEncoderTransform.java

```diff
@@ -40,7 +40,7 @@ public ValFrame apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
     Frame[] encodingMapFrames = getEncodingMapFrames(env, stk, asts);
     Frame frame = getFrameToTransform(env, stk, asts);
     String[] teColumnsToEncode = getTEColumns(env, stk, asts);
-    byte dataLeakageHandlingStrategy = getDataLeakageHandlingStrategy(env, stk, asts);
+    TargetEncoder.DataLeakageHandlingStrategy dataLeakageHandlingStrategy = TargetEncoder.DataLeakageHandlingStrategy.fromVal(getDataLeakageHandlingStrategy(env, stk, asts));
```

Comment:
inside getDataLeakageHandlingStrategy you are doing getVal() and here you construct the enum again using fromVal, seems unnecessary

Reply:
This code has been removed only - FYI. I'm not the author. I probably have to ask the author for permission to change it :) Will do :)

Reply:
Resolved, thanks.